ha/resource/sspl

```diff
@@ -118,45 +118,40 @@ stateful_check_state() {
 start_sspl_service() {
         systemctl start sspl-ll
         echo "START (master): $?"
-        count=0
-        while true
-        do
-            status=`systemctl is-active sspl-ll`
-            if [[ $status == "active" ]]; then
-                return 0
-            elif [[ $status == "failed" ]]; then
-                return 1
-            elif [[ $status == "unknown" ]]; then
-                ((count++))
-                if [ $count == 5 ]; then
+
+        #check status
+        for retry in {1..5}; do
+            status=$(systemctl is-active sspl-ll)
+            case $status in
+                active )
+                    return 0
+                    ;;
+
+                failed )
                     return 1
-                fi
-            fi
+                    ;;
+
+                * )
```

Comment:
Default option "\* )" can be removed as there is no action taken for the same.

Reply:
Yes, We can remove it